.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please provide the following details along with the issue description - Thank you for the contribution.
+
+* `node` version is: `node --version` would log the version
+* `npm` version is: `npm --version` would log the version
+* `webpack` version is: the value next to `version` key at `./node_modules/webpack/package.json`
+* `webpack-validator` version is: the value next to `version` key at `./node_modules/webpack-validator/package.json`
+* `OS` details: http://whatsmyos.com/ would provide the details
+* error log:
+
+Any other details, that might be helpful to reproduce the issue.
```

Comment:
@jonathanewerner the main intention is to provide a base template for people to provide information.

>> but issues are also for features and ideas

Absolutely, consider this PR as the initiation of discussion regarding these templates.
Please share your thoughts about what needs to be included/excluded.

I guess you can even push commits to this PR. Feel free in what ever is more convenient.

Anyway, it would great to have these templates, if I may say so :smile: